packages/components/divider/src/divider.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div :class="['el-divider', `el-divider--${direction}`]">
+  <div
+    :class="['el-divider', `el-divider--${direction}`]"
+    :style="{ '--el-border-style': $props.lineStyle }"
```

Comment:
I think the `$props` prefix is unnecessary since if you don't have the same API returned by setup function, you should always be good to use `lineStyle` in the template directly.